Katello/katello
# Fixes #36829 - supprimer used incorrectly in French translations

#### What are the changes introduced in this pull request?
Updates all translations to be up-to-date with Transifex. Also fixes the wrong translations noted in https://projects.theforeman.org/issues/36829 and https://projects.theforeman.org/issues/36945

#### Considerations taken when implementing this change?

...

#### What are the testing steps for this pull request?
1) Switch to French
2) Check that the "Create Activation Key" button says "Creer une cle d'activation" (with accent marks)
3) Check that the "Create Product" button says "Creer un produit" (with accent marks)
